src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,77 @@
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { useSearchParams } from 'react-router-dom';
+import { filterPeople } from '../utils/filterPeople';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [peopleLoadError, setPeopleLoadError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const [searchParams] = useSearchParams();
+  const sex = searchParams.get('sex') || '';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setPeopleLoadError('Something went wrong'))
```

Comment:
Consider providing a more descriptive error message or logging the actual error to help with debugging. For example, you could log the error to the console: `.catch(error => { console.error(error); setPeopleLoadError('Something went wrong'); })`.